### ISSUE TITLE
[bug]: V2.9.44 Startlogo „MonsterVan“ + Hintergrund + Bild lässt sich nicht ändern

### ISSUE THREAD

**Issue by MK-66** [ioBroker/ioBroker.vis-2#405]
### No existing issues.

- [X] There is no existing issue for my problem.

### Describe the bug

Mit der neuen Version V2.9.44 wird beim Laden der Runtime anstelle des gewohnten ioBroker Logos nun ein „MonsterVan“ Logo angezeigt. Außerdem funktionieren die neuen Einstellmöglichkeiten bezüglich „Hintergrundfarbe“ und „Hintergrundbild verwenden“ nicht.

### To Reproduce

VIS-2 Instanzeinstellungen: 
Eine selbst gewählte Hintergrundfarbe wird beim Start der Runtime nur dann kurz angezeigt, wenn auch die Checkbox „Logo ausblenden“ nicht markiert ist, ansonsten bleibt der Starthintergrund einfach weiß. Ein Hintergrundbild wird, egal welche Checkboxen aktiviert oder deaktiviert sind, beim Start der Runtime oder des Editors nicht angezeigt.

Die Checkbox „Logo ausblenden“ funktioniert insofern, als das bei deaktivierter Checkbox auch kein Logo bzw. das „MonsterVan“ Logo nicht angezeigt wird, sondern nur ein leerer weißer Bildschirm bis die Runtime vollständig geladen ist.

### Expected behavior

Standardmäßig sollte beim Start wieder das gewohnte drehende VIS-2 ioBroker Logo angezeigt werden, statt des „MonsterVan“ Logos. Die in den Instanzeinstellungen gewählte Hintergrundfarbe und/oder Hintergrundbild sollte auch funktionieren.

### Screenshots & Logfiles

**Screenshot „MonsterVan“ Logo**
![MonsterVanStartlogo](https://github.com/ioBroker/ioBroker.vis-2/assets/56516310/32cfa01c-a55f-457b-8643-4d90939909b9)

**Screenshot Instanzeinstellungen**

![VisInstanzeinstellungen](https://github.com/ioBroker/ioBroker.vis-2/assets/56516310/2ef76ca3-6202-4fcf-b79c-86d7267eaf14)

### Adapter version

2.9.44

### js-controller version

5.0.19

### Node version

18.19.1

### Operating system

Debian 12

**Comment by mcm1957** [comment]
duplicate #404 ?

**Comment by MK-66** [comment]
@mcm1957: Ja, ist teilweise ähnlich oder doppelt zu issue 404. Habe beim Erstellen meines issues zu spät gesehen, dass es das issue 404 von afuerhoff schon gab, sonst hätte ich meine Beschreibungen bei seinem issue 404 hinzugefügt. Ich denke aber, dass ich in meinem issue 405 noch ein ein paar mehr Fehler bezüglich der nicht funktionierenden neuen Einstellungen in der VS 2.9.44 Instanz  detaillierter beschrieben habe.

**Comment by MK-66** [comment]
Mit V2.9.47 ist soweit wieder alles ok, allerdings funktioniert "Hintergrundbild verwenden" noch nicht, dies ist mir persönlich aber auch nicht so wichtig.

**Comment by seb2010** [comment]
Das Logo ist wieder das alte. Die Hintergrundfarbe des Ladescreens wird aber immernoch nicht gesetzt und auch das Logo ausblenden funktioniert noch nicht.
Im Missverständnisse zu vermeiden: ich meine den Ladescreen, welcher beim ersten Laden oder Neuladen (F5) der Visualisierung mittels index.html angezeigt wird. Ich meine nicht den Reload-Screen bei Verbindungsverlust, den man (inkonsistenter Weise) im Editor einstellen kann.

**Comment by seb2010** [comment]
Ich hatte bei eigenen Debug-Versuchen gesehen, dass im Code @@loginBackgroundColor@@ statt, wie glaube ich gewollt @@**loading**BackgroundColor|@@ steht.
Das ist immernoch der Fall im customWidgets.js, welches geladen wird. Und auch in den /static/js/main.????.js ist das noch so geschrieben...
